testing.md

```diff
@@ -0,0 +1,71 @@
+# Testing
+
+## Why write tests?
+
+We write automated tests to allow us to quickly demonstrate that our software meeting business requirements. They're one of a few tools we use to increase our confidence in our work prior to deployment.
```

Comment:
Whilst I don't disagree with the statement "we write tests to stop other people breaking our stuff" do I disagree with the idea that "we don't write tests to prove our code does what we say it does". Writing tests like this doesn't say "we don't trust people to do their job", it says "we understand that people are human and make mistakes", automated testing is a way to catch some of these mistakes.

I don't think there is really a huge difference between these two ways of thinking about it though. If you manage to break the application... then it no longer behaves as expected. Both are a valuable ways of thinking about how to write tests.

In any event I think the latest version reflects both of these.